doc/task-definition-example.yml

```diff
@@ -32,4 +32,20 @@ properties:
   - property_file: ../test/tasks/other.prp
     expected_verdict: true
 
+# Optional, contains dict with key-value pairs that
+# provide additional information about the context of the task definition.
+# If given, benchexec will pass this information to the tool-info module.
+#
+# Keys and their values are not interpreted by benchexec and may differ between application areas.
+# Because of this, they should be namespaced to avoid naming conflicts
+# and put their intended meaning in context (no pun intended; see below for an example).
+context:
+  # Example field for this dict.
+  # Machine model the task definition is defined for.
+  # Properties of this task definition are only guaranteed to be
+  # valid on the input_files for the machine model specified here.
+  #
+  # Values tools are expected to support: linux32 and linux64
+  - com.github.sosy-lab.sv-benchmarks.machine_model: linux32
```

Comment:
Just noticed that you seem to use a list of dicts inside `context`. Is that intentional?

Reply:
Nice catch. No, definitely not intentional. Each key should only occur once - the comment described this correctly, but I "mistyped" in the example. Fixed now.